src/stories/CardPassword.stories.tsx

```
@@ -0,0 +1,30 @@
+
+export const Primary: Story = {
+  args: {
+    handleChange: () => {},
+    handleStep: () => {},
+    step: 4,
+    password: "12",
+    errorMessage: "",
+  },
+};
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)